### PR TITLE
131 document next mind

### LINKED ISSUE #131: Document NextMind

### PR DESCRIPTION
**Pull request by FergusKidd**
resolves #131

### CODE REVIEW

File: docs/NextMind-setup.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+This document covers how to setup the NextMind Unity app and server on the stretch to enable mind-controlled operation of the stretch robot.
+
+## Stretch server
+
+Move the 'nextmind-stretch-server' folder to the stretch robot directory, and ensure you have the control agent fiels also loaded onto the stretch. Ensure the path commands variable in [app.py](./nextmind-stretch-server/app.py) reflect the install location of the python 2.7 motion scripts if using a different location than "/home/hello-robot/Chatbot"
+
+`N.B. These scripts will be replaced as soon as a python 3 module is available for robot move functionality`
+
+To start the server on the stretch, ensure the correct libraries are installed according to 'requirements.txt'.
```
Comment (Sealjay-clj) [inline docs/NextMind-setup.md:11]:
```suggestion
To start the server on the stretch, ensure the correct libraries are installed according to `requirements.txt`.
```

File: docs/NextMind-setup.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Unity
+
+Currently, the url needs to be given to unity manually. Update line 119 of [MoveFunctions.cs](./nextmind-robot-telepresence/Assets/Scripts/MoveFunctions.cs) with the ngrok url:
+
+```
+using (UnityWebRequest www = UnityWebRequest.Post($"[server.path.here]/{method}/{ammount}", form))
+```
+
+Next, run the unity app by opening the [nextmind-robot-telepresense](./nextmind-robot-telepresence) directory in the unity explorer.
```
Comment (Sealjay-clj) [inline docs/NextMind-setup.md:48]:
```suggestion
Next, run the unity app by opening the [nextmind-robot-telepresence](./nextmind-robot-telepresence) directory in the unity explorer.
```

File: nextmind-stretch-server/app.py [hunk 3]
```diff
@@ -4,42 +4,84 @@
 from starlette.responses import JSONResponse
 from starlette.routing import Route
 
-path_to_commands = "/home/hello-robot/Chatbot/moveGrip.py"
+path_to_commands = "/home/hello-robot/Chatbot"
```
Comment (Sealjay-clj) [inline nextmind-stretch-server/app.py:7]: This should be capitalised as a constant

File: nextmind-stretch-server/app.py [hunk 4]
```diff
@@ -4,42 +4,84 @@
...
 
 
 async def open_grip(request):  # scan:ignore
     move_amount = request.path_params["amount"]
-    os.system("python " + path_to_commands + " 'open' " + str(move_amount))
+    os.system("python " + path_to_commands + "/moveGrip.py 'open' " + str(move_amount))
```
Comment (Sealjay-clj) [inline nextmind-stretch-server/app.py:12]: This is what I was saying should be a function - I didn't mean truncate path_to_commands

**Review by Sealjay-clj — COMMENTED** [review]
I've reviewed it - but it looks like only some of the changes have been made; if you're going to do them now, I'd move the os call into a function, in case we go ahead and integrate ROS - or use Rocos as planned.

File: nextmind-stretch-server/app.py (hunk 4 above)
Comment (FergusKidd) [reply nextmind-stretch-server/app.py:12]: changed to run_command("moveGrip",move_amount), for all commands

Comment (FergusKidd) [reply nextmind-stretch-server/app.py]: done

**Pull request merged**